### ISSUE TITLE
Adding a delay to an action after capturing a screenshot?

### ISSUE THREAD

**Issue by ireun** [ShareX/ShareX#1712]
Hello, I'm posting the screenshots that I'm taking into conversation in Facebook. ShareX is first copying to image to clipboard, than uploads it, and than, after, upload, it's copying the URL to clipboard. At least that's how it is set for me.

The thing is, if the picture is in the clipboard, i can just "Ctrl + V" it into the conversation in Facebook, but sometimes I'm not fast enough to do this, and I'm pasting the URL instead of the image. Of course I don't want that. ;)

I though, if there would be a way, to set a delay before copying the URL to clipboard, I would have more time to paste the image, while I would still able to paste the URL if needed, just seconds later. 

Hoping for You to implement this idea @Jaex! ;)

**Comment by Jaex** [comment]
I don't liked your idea, it is very specific to your use cases. What if people want to have delay for each after capture tasks separately? How am I supposed to make ui for such thing?

**Comment by ireun** [comment]
Right, that might be a problem.. So maybe you could add "Copy last URL" as a task, not only "after upload", I would than assign it to one of my spare mouse key, as a workaround.

**Comment by Aebian** [comment]
Hmm after reading the entire conversation I came across a couple of things:

- Maybe make an option to press a specific button after capturing a screenshot to upload and then get the link.
- Or upload it directly (as it is right now) and then press a button to get the url.

Since sometimes I just want the captured screen rather then upload it and get the link.
If there would be such option that would be nice and I think many ShareX users would like that too =)

**Comment by ireun** [comment]
I also was thinking about the 2nd option, would work for me.. ;)

**Comment by gergy008** [comment]
I have a separate hotkey to upload my clipboard and it works so much better in my use case.

**Comment by Knicey** [comment]
Yeah I also have use for this kind of functionality.
A workaround I've found is to edit my notification interactions.
I leave the default actions of capture -> copy to clipboard -> upload -> copy image url
but I set my "Left click action" to "Copy image"
![image](https://user-images.githubusercontent.com/45131167/147939850-b0009c7b-066b-4e0b-806c-4e02cc67e829.png)
